README.md

```diff
@@ -50,6 +50,7 @@ Have any feedback or questions? [Create a discussion](https://github.com/TwiN/ga
     - [Configuring Teams alerts](#configuring-teams-alerts)
     - [Configuring Telegram alerts](#configuring-telegram-alerts)
     - [Configuring Twilio alerts](#configuring-twilio-alerts)
+    - [Configuring Matrix alerts](#configuring-matrix-alerts)
```

Comment:
Could you move it up a bit so it's sorted alphabetically? (excl. the `custom` alert, which should stay at the bottom)